client/src/styles/fluent.scss

```diff
@@ -68,7 +68,7 @@ $fluent-color-localised: rgb(102, 15, 242);
     background-color: #368b39;
     background-image: url("../images/menu-icon-24-white.png");
     background-position: 6px center;
-    background-repeat: none;
+    background-repeat: no-repeat;
```

Comment:
`none` is [not a valid value for this property](https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat) - this was the main cause of https://github.com/silverstripe/.github/issues/332#issuecomment-2611331639